pkg/addons/clean.go

```diff
@@ -162,6 +158,9 @@ func handleSpecialObjects(existing, o unstructured.Unstructured) {
 			o.Object["spec"].(map[string]interface{})["clusterIP"] = existing.Object["spec"].(map[string]interface{})["clusterIP"]
 			o.Object["spec"].(map[string]interface{})["externalTrafficPolicy"] = existing.Object["spec"].(map[string]interface{})["externalTrafficPolicy"]
 			o.Object["spec"].(map[string]interface{})["ports"] = existing.Object["spec"].(map[string]interface{})["ports"]
+			return
```

Comment:
Why not remove this and also remove the clusterIP copy above?

Reply:
ack :)